semgrep/Pipfile

```diff
@@ -14,6 +14,3 @@ xmldiff = "*"
 
 [packages]
 semgrep = {editable = true,path = "."}
-
-[requires]
-python_version = "3.7"
```

Comment:
or can we require > 3.6 or something?

Reply:
I was just looking into this. Apparently the answer is "no": https://github.com/pypa/pipenv/issues/1050 :-1: